.travis.yml

```diff
@@ -16,6 +13,12 @@ cache:
 
 jobs:
   include:
+    - stage: coverage
+      node_js:
+        - 10
+        - 8
```

Comment:
why do you want to run coverage twice?

Reply:
We currently do already, see #508. It seems as if coveralls recently has become smart enough to ignore the duplicate (check more recent PRs)

Reply:
In the future I could envision being smart and trying to avoid this, but I prefer keeping the PRs modular and do the jest migration first.

Reply:
so like no travis config change?

Reply:
I changed travis because jest was slightly slower (it can be a number of reasons, really), so I wanted to avoid what we previously did, which was run tests, then run tests _again_ for coveralls.

Reply:
hm. I believe jest generate coverage report without running tests if you ran tests before

Reply:
Yeah, I tried this with mocha before switching to jest, but I couldn't seem to access the coverage file then from the `after_script` or `after_success`, where coveralls was run. 🤔